### PR TITLE
DC-6881 Update as per content style guide for apostrophe

### PR DESCRIPTION
**Pull request by kjainsc**

### CODE REVIEW

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/childcare/childcareTaxFreeDevolved_B.scala.txt [hunk 1]
```diff
@@ -7,10 +7,10 @@ Apply now for Tax-Free Childcare!
...

Thank you for your interest in the childcare service.

Yours faithfully,
+
You can find more information on Tax-Free Childcare and check if you’re eligible at: www.childcarechoices.gov.uk

Thank you for your interest in the childcare service.

Yours faithfully,
```
Comment (ardwalker) [inline app/uk/gov/hmrc/hmrcemailrenderer/templates/childcare/childcareTaxFreeDevolved_B.scala.txt:13]: As you are changing the text, is it worth adding a space at the end of those sentences?
Just a thought.

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/dfs/dfsSubmissionConfirmationEmailCIS.scala.html [hunk 2]
```diff
@@ -17,11 +17,11 @@
...
     <p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">Dear @{params("name")},</p>
 
-    <p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">We've received your application to register as a sub-contractor in the construction industry.</p>
+    <p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">We’ve received your application to register as a sub-contractor in the construction industry.</p>
 
     <p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">Your reference is @{params("submissionReference")}.</p>
 
-    <p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">We'll write to you about your request and let you know if it has been accepted. If we need any further details, we'll contact you by letter or phone. We'll never ask you for your details by email.</p>
+    <p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">Well write to you about your request and let you know if it has been accepted. If we need any further details, well contact you by letter or phone. Well never ask you for your details by email.</p>
```
Comment (ardwalker) [inline app/uk/gov/hmrc/hmrcemailrenderer/templates/dfs/dfsSubmissionConfirmationEmailCIS.scala.html]: This is incorrect

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/dfs/dfsSubmissionConfirmationEmailCIS.scala.txt [hunk 3]
```diff
@@ -2,11 +2,11 @@
 
 Dear @{params("name")},
 
-We've received your application to register as a sub-contractor in the construction industry.
+We’ve received your application to register as a sub-contractor in the construction industry.
 
 Your reference is @{params("submissionReference")}.
 
-We'll write to you about your request and let you know if it has been accepted. If we need any further details, we'll contact you by letter or phone. We'll never ask you for your details by email.
+We’ll write to you about your request and let you know if it has been accepted. If we need any further details, we’ll contact you by letter or phone. Well never ask you for your details by email.
```
Comment (ardwalker) [inline app/uk/gov/hmrc/hmrcemailrenderer/templates/dfs/dfsSubmissionConfirmationEmailCIS.scala.txt]: Error Well should be We'll

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/dfs/dfsSubmissionConfirmationEmailCIS.scala.html (hunk 2 above)
Comment (kjainsc) [reply app/uk/gov/hmrc/hmrcemailrenderer/templates/dfs/dfsSubmissionConfirmationEmailCIS.scala.html]: yes, updated now. thanks

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/dfs/dfsSubmissionConfirmationEmailCIS.scala.txt (hunk 3 above)
Comment (kjainsc) [reply app/uk/gov/hmrc/hmrcemailrenderer/templates/dfs/dfsSubmissionConfirmationEmailCIS.scala.txt]: yes, updated now. thanks

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/dfs/dfsSubmissionConfirmationEmailEmptyTurnAroundTime.scala.html [hunk 4]
```diff
@@ -34,7 +34,7 @@
     }.</p>
 }
 @if(params.get("warningInformation").exists(_ == "true")) {
-<p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">If we need any further details, we'll contact you by letter or phone. We may ask you to sign in to a service to provide more details but we won't ask for your personal details by email.</p>
+<p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">If we need any further details, well contact you by letter or phone. We may ask you to sign in to a service to provide more details but we wont ask for your personal details by email.</p>
```
Comment (ardwalker) [inline app/uk/gov/hmrc/hmrcemailrenderer/templates/dfs/dfsSubmissionConfirmationEmailEmptyTurnAroundTime.scala.html]: incorrect

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/dfs/dfsSubmissionConfirmationEmailEmptyTurnAroundTime.scala.txt [hunk 5]
```diff
@@ -9,7 +9,7 @@ Your reference is @{params("submissionReference")}.
 
 @if(params.isDefinedAt("paragraphOne")) {@{params("paragraphOne")}.}
 @if(params.isDefinedAt("paragraphTwo")) {@{params("paragraphTwo")}.}
-@if(params("warningInformation").equals("true")) {If we need any further details, we'll contact you by letter or phone. We may ask you to sign in to a service to provide more details but we won't ask for your personal details by email.} 
+@if(params("warningInformation").equals("true")) {If we need any further details, well contact you by letter or phone. We may ask you to sign in to a service to provide more details but we wont ask for your personal details by email.}
```
Comment (ardwalker) [inline app/uk/gov/hmrc/hmrcemailrenderer/templates/dfs/dfsSubmissionConfirmationEmailEmptyTurnAroundTime.scala.txt]: Incorrect

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/emachelpdesk/emacHelpdeskFailureConfirmationEmail.scala.txt [hunk 6]
```diff
@@ -1,8 +1,8 @@
-@(params: Map[String, Any])You've submitted a document to HMRC for @params("service_name")
+@(params: Map[String, Any])You’ve submitted a document to HMRC for @params("service_name")
 
 Dear @params("full_name"),
 
-We've received a document that you've submitted, however the submission wasn't successfully processed.
+We’ve received a document that you’ve submitted, however the submission wasn't successfully processed.
```
Comment (ardwalker) [inline app/uk/gov/hmrc/hmrcemailrenderer/templates/emachelpdesk/emacHelpdeskFailureConfirmationEmail.scala.txt:5]: A bit pedantic, but the ' in wasn't is still the old style

**Review by ardwalker — APPROVED** [review]

**Pull request merged**